samples/bluetooth/mesh/light/child_image/hci_ipc.conf

```diff
@@ -1,6 +1,26 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
 # Copy controller configuration from prj.conf
 CONFIG_BT_CTLR_DUP_FILTER_LEN=0
 CONFIG_BT_CTLR_LE_ENC=n
 CONFIG_BT_CTLR_CHAN_SEL_2=n
 CONFIG_BT_CTLR_MIN_USED_CHAN=n
 CONFIG_BT_CTLR_PRIVACY=n
+
+# One extra connection for mesh GATT/proxy and one for SMP BT.
+CONFIG_BT_MAX_CONN=3
+CONFIG_BT_CTLR_SDC_PERIPHERAL_COUNT=3
+
+# Enable extended advertising
+CONFIG_BT_EXT_ADV=y
```

Comment:
Isn't this redefined via overlay?
Here: https://github.com/nrfconnect/sdk-nrf/blob/main/samples/CMakeLists.txt#L138-L146
and here: https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/bluetooth/mesh/hci_ipc_child_image_overlay.conf#L1-L12
How does it work?